pkg/agent/agent.go

```diff
@@ -539,13 +547,26 @@ func (c *agentController) SyncAddOnPlacementScore(ctx context.Context) error {
 			c.log.Error(err, fmt.Sprintf("failed to update the addOnPlacementScore status in %s", c.clusterName))
 			return err
 		}
-	}
 
-	c.log.Info(fmt.Sprintf("updated the addOnPlacementScore for %s: %v", c.clusterName, len(hcList.Items)))
+		c.log.Info(fmt.Sprintf("updated the addOnPlacementScore for %s: %v", c.clusterName, len(hcList.Items)))
```

Comment:
Personal preference only, due to the repeated use of `len(hcList.Items)` below and easier to understand if this is assign to a var like `hcCount` or something like that.

Reply:
yes